modules/kafka/kafka.go

```diff
@@ -125,12 +152,78 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 	return &KafkaContainer{Container: container, ClusterID: clusterID}, nil
 }
 
-func WithClusterID(clusterID string) testcontainers.CustomizeRequestOption {
-	return func(req *testcontainers.GenericContainerRequest) error {
-		req.Env["CLUSTER_ID"] = clusterID
+func trimListeners(listeners []KafkaListener) {
```

Comment:
Do you think we can embed this function into the validate listeners one? That will avoid having two steps in the call, and I also consider this code part of the validation

Reply:
I guess you are right